src/cmds/scripts/pbs_snapshot

```diff
@@ -35,20 +35,52 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
-pbs_conf=${PBS_CONF_FILE:-/etc/pbs.conf}
-if [ -r "${pbs_conf}" ]; then
-	. ${pbs_conf}
+ptllibpath=
+ptlbinpath=
+
+if [ -f "/etc/debian_version" ]; then
+        ptl_prefix_lib=$( dpkg -L pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null )
+else
+        ptl_prefix_lib=$( rpm -ql pbspro-ptl 2>/dev/null | grep -m 1 lib$ 2>/dev/null )
+fi
+if [ "x${ptl_prefix_lib}" != "x" ]; then
+        python_dir=$( /bin/ls -1 ${ptl_prefix_lib} )
```

Comment:
Is it common practice to put spaces after and before your parentheses?

Reply:
I actually just copied the code from ptl.csh, I'll remove the spaces